demo/commerce/play-shop-discover-to-ordercloud-feed-importer/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
Could you please look at our other projects eslint configuration and try to use most of the same rules for this project as well if possible. Things that result in an error in other projects should also result in an error in this project (unless there is a good reason it should not). It would ensure a similar code quality across all projects of the repository.